app/models/edition/has_mainstream_categories.rb

```diff
@@ -31,6 +31,15 @@ def can_be_associated_with_mainstream_categories?
     true
   end
 
+  def remove_mainstream_category(category)
```

Comment:
I think this is largely fine, however using `update_attribute` in a model isn't ideal. To discourage use of this method without good reason perhaps put a bang on it (`remove_mainstream_category!`) and add a comment explaining that its intended to be used in data migrations, and that it's included here because it's expected to be used a lot over the coming months as taxonomy on GOV.UK evolves.